src/components/heatmap/HeatmapTooltipSubscriber.js

```diff
@@ -19,60 +19,56 @@ export default function HeatmapTooltipSubscriber(props) {
   const [x, setX] = useState(null);
   const [y, setY] = useState(null);
 
+  const [{ cellHighlight, geneHighlight }] = useCoordination(['cellHighlight', 'geneHighlight'], coordinationScopes);
+
   useEffect(() => {
-    const cellsHoverToken = PubSub.subscribe(
-      CELLS_HOVER, (msg, hoverInfo) => {
-        if (!hoverInfo) {
-          setCellInfo(null);
-          setSourceUuid(null);
-        } else {
-          const newCellInfo = getCellInfo(hoverInfo.cellId);
-          setCellInfo(newCellInfo);
-          setSourceUuid(hoverInfo.uuid);
-          if (viewInfo && viewInfo.project) {
-            const [newX, newY] = viewInfo.project(hoverInfo.cellId, null);
-            if (transpose) {
-              setX(newX);
-            } else {
-              setY(newY);
-            }
-          }
-        }
-      },
-    );
-    const genesHoverToken = PubSub.subscribe(
-      GENES_HOVER, (msg, hoverInfo) => {
-        if (!hoverInfo) {
-          setGeneInfo(null);
-          setSourceUuid(null);
-        } else {
-          const newGeneInfo = getGeneInfo(hoverInfo.geneId);
-          setGeneInfo(newGeneInfo);
-          setSourceUuid(hoverInfo.uuid);
-          if (viewInfo && viewInfo.project) {
-            const [newX, newY] = viewInfo.project(null, hoverInfo.geneId);
-            if (transpose) {
-              setY(newY);
-            } else {
-              setX(newX);
-            }
-          }
-        }
-      },
-    );
     const viewInfoToken = PubSub.subscribe(
```

Comment:
I'm not sure what the solution here is, but this feels out of place given how clean everything else is.  Maybe in the docs or something we should note that tooltip events are somewhat outside of the scope of global state?

Reply:
I have cleaned up the tooltips a little bit. They still feel a little odd but I can make an issue.